dist/extension.js

```diff
@@ -0,0 +1,47 @@
+const vscode = require('vscode');
+
+const THEME_CONFIGURATION_ID = 'markdown-preview-github-styles';
+const THEME_CONFIGURATION_KEY = 'colorMode';
+
+const THEME_CONFIGURATION_VALUES = {
+    'auto': true,
+    'system': true,
+    'light': true,
+    'dark': true,
+    'high-contrast': true
```

Comment:
Is `high-contrast` still a valid value here?

Reply:
Not yet, I don't think this will be possible without a second PR :(